CaptureSDK/GiniCaptureSDK/Tests/GiniCaptureSDKTests/ImageMetaInformationManagerTests.swift

```diff
@@ -62,5 +62,42 @@ final class ImageMetaInformationManagerTests: XCTestCase {
         XCTAssert((mutableInformation.getMetaInformation(forKey: key) as? String)?.contains("GiniCaptureVer") == true,
                   "filtered data did not set custom fields")
     }
+    
+    func testSettingDefaultEntryPointToMetaInformation() {
+        guard let mutableInformation = manager.metaInformation?.mutableCopy() as? NSMutableDictionary else {
+            return XCTFail("failed to retrieve mutable meta information from test data")
+        }
+        let value = "button"
+        let key = kCGImagePropertyTIFFMake as String
+        mutableInformation.set(metaInformation: value as AnyObject?, forKey: key)
+        XCTAssert(mutableInformation.getMetaInformation(forKey: key) as? String == value,
+                  "failed to set value for entry point correctly on meta information")
+    }
+    
+    func testSettingFieldEntryPointToMetaInformation() {
+        guard let mutableInformation = manager.metaInformation?.mutableCopy() as? NSMutableDictionary else {
+            return XCTFail("failed to retrieve mutable meta information from test data")
+        }
+        let value = "field"
+        let key = kCGImagePropertyTIFFMake as String
+        mutableInformation.set(metaInformation: value as AnyObject?, forKey: key)
+        XCTAssert(mutableInformation.getMetaInformation(forKey: key) as? String == value,
+                  "failed to set value for entry point correctly on meta information")
+    }
+    
+    func testFieldEntryPointToMetaInformation() {
+        let metaManager = ImageMetaInformationManager(imageData: GiniCaptureTestsHelper.fileData(named: "testFieldEntryPoint", fileExtension: "jpg")!, imageSource: .camera)
+        let metaInformation = metaManager.metaInformation as? NSMutableDictionary
+        let value = "field"
+        let existingUserComment = metaInformation?.getMetaInformation(forKey: kCGImagePropertyExifUserComment as String)
+        let components = existingUserComment?.components(separatedBy: ",")
+        let userCommentComponent = components?.filter({ (component) -> Bool in
+            return component.contains("EntryPoint")
+        })
+        let equasionComponents = userCommentComponent?.last?.components(separatedBy: "=")
+        let entryPoint = equasionComponents?.last
+        XCTAssert(entryPoint == value,
+                  "failed to set value for entry point correctly on meta information")
+    }
         
```

Comment:
[Request] Would be great if you could add two tests (one for button and one for field entry point) with which you check that the "EntryPoint" is added to jpegs when using `imageByAddingMetadata()`. This is important I think because this is how we add "EntryPoint" to jpegs from the camera or using upload.